components/content/content-editor/content-editor.ts

```diff
@@ -253,6 +253,7 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 				'data-prevent-shortkey': '',
 			},
 		});
+		this.updateIsEmpty(state);
```

Comment:
this is that unrelated fix you cherry picked to fix the placeholder showing up in game descriptions on load?

Reply:
yep